### DISCUSSION TITLE
issue: AJV resolvers custom keywords errors are not present in formState.errors

### DISCUSSION THREAD

**Discussion by javier-brenes-zen** [react-hook-form/react-hook-form#11899] [Q&A]
### Version Number

7.51.0

### Codesandbox/Expo snack

https://stackblitz.com/edit/stackblitz-starters-vtxdst?file=src%2FApp.js,src%2FcustomKeywords.js

### Steps to reproduce

1. on stackblitz project go to customKeywords, there you will see `cannotBeEqualTo` definition and verify we are passing error.
2. on inputs ( browser side ) add `CR` for state, and `123` for Plate
3. you will noticed form is not valid due to `cannotBeEqualTo` validations
4. check `errors` in console, I am expecting to see the error `cannotBeEqualTo` custom keyword produces but it is not there so I cannot give feedback to user.

### Expected behaviour

I am expecting getting the error in `formState.errors` so I can use it to give feedback to the user, I already tested the Json schema using ajv directly and I am seeing the error being returned so it has to be something in React-hook-form that deletes the error or doesn't include it on the `formState.errors`.

AJV validation codesandbox: https://shhttps://shorturl.at/fgJQ3orturl.at/fgJQ3

Evidence: 
I added some console.log at https://github.com/react-hook-form/resolvers/blob/master/ajv/src/ajv.ts locally so I guarantee error is being captured

![Screenshot 2024-05-15 at 4 33 14 PM](https://github.com/react-hook-form/react-hook-form/assets/68287257/ea2231d3-2b80-47cd-8887-039f236f77b9)

and just after that this is what I get from react-hook-form on my app.

![Screenshot 2024-05-15 at 4 37 15 PM](https://github.com/react-hook-form/react-hook-form/assets/68287257/92176172-0fa8-4ce2-94d5-bfc94900345b)

as you can see form is `invalid` but no errors come.

### What browsers are you seeing the problem on?

Chrome

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by bluebill1049** [comment]
Pretty sure this is a lack of input registration, have you tired to register those missing error field?

**Reply by javier-brenes-zen** [reply]
the error validation is at `root` level, not sure how to register that part.

as an example imagine I have this `object`: 
```
{ atl: { plate: string, value: string } }
```

`plate` and `value` are inputs they are registered, but if the error is at `atl` ( based on ajv configuration ) how can we registered that ?
